package.json

```diff
@@ -122,6 +130,31 @@
                 "mac": "Alt+cmd+o",
                 "when": "editorTextFocus"
             }
-        ]
+        ],
+        "menus": {
+            "editor/context": [
+                {
+                    "command": "clangd.typeHierarchy",
+                    "when": "resourceLangId == cpp",
+                    "group": "0_navigation@4"
```

Comment:
What's this, you ask? This is how I got it to be placed in the [references section](https://github.com/microsoft/vscode-references-view/blob/f63eaed9934ca5ecc8f3fb3ca096f38c6e5e181f/package.json#L162).

I'd write this in a comment, but... JSON doesn't support comments.

Reply:
you might get away with adding a "_comment" key, but up to you whether that's too horrible